### PR TITLE
New version: Bitnami.WAMP version 8.1.7-1

### PR DESCRIPTION
**Pull request by russellbanks**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/66325)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by OfficialEsco** [comment]
@denelon it appears this one failed because it didn't give it enough time to install
InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20220714133806

Probably the same with #66324 and #66323 if its the same installer

**Comment by yao-msft** [comment]
By looking at the other 2 failures, both are stuck at launch time when "Allow network access type" window is popped for user input.
I guess this is related to `Custom: --launch_cloud 0` switch.

File: manifests/b/Bitnami/WAMP/8.1.7-1/Bitnami.WAMP.installer.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+  Silent: --mode unattended --unattendedmodeui none
+  SilentWithProgress: --mode unattended --unattendedmodeui minimalWithDialogs
+  Interactive: --mode qt
+  InstallLocation: --prefix <INSTALLPATH>
+  Log: --debugtrace <LOGPATH>
+  Custom: --launch_cloud 0
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.bitnami.com/files/stacks/wampstack/8.1.7-1/bitnami-wampstack-8.1.7-1-windows-x64-installer.exe
+  InstallerSha256: A768518C9447DD9C9B7B9829CEB0B621CD195917A6A03DA4069682E3D29C6377
```
Comment (ItzLevvie) [inline manifests/b/Bitnami/WAMP/8.1.7-1/Bitnami.WAMP.installer.yaml:18]:
```suggestion
  InstallerSha256: A768518C9447DD9C9B7B9829CEB0B621CD195917A6A03DA4069682E3D29C6377
  AppsAndFeaturesEntries:
  - DisplayName: WAMP packaged by Bitnami
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**